loaders/standard-loader.js

```diff
@@ -0,0 +1,7 @@
+const linter = require('typescript-standard')
+
+module.exports = function standardLoader (text) {
+  this.cacheable()
```

Comment:
https://webpack.github.io/docs/how-to-write-a-loader.html#flag-itself-cacheable-if-possible